beta/nncf/tensorflow/graph/converter.py

```diff
@@ -274,47 +303,53 @@ def _get_nncf_graph_from_sequential(model: tf.keras.Model) -> NNCFGraph:
                      in_ports=[0],
                      out_ports=[0],
                      is_shared=False)
+
+        module_attributes = None
         if layer_metatype in GENERAL_CONV_LAYER_METATYPES:
-            module_attributes = _get_module_attributes(model_layer)
+            module_attributes = _get_module_attributes(model.get_layer(layer_name))
             attrs.update({NNCFGraph.MODULE_ATTRIBUTES: module_attributes})
 
-        nncf_graph.add_node(layer_name, **attrs)
-        if producer_layer is not None:
-            input_shape = _prepare_shape(_get_layer(model, layer_name).input_shape)
-            attr = {
-                NNCFGraph.ACTIVATION_SHAPE_EDGE_ATTR: input_shape[0],
-                NNCFGraph.IN_PORT_NAME_EDGE_ATTR: 0
-            }
-            nncf_graph.add_edge(producer_layer, layer_name, **attr)
-        producer_layer = layer_name
-
-    output_model_layer = model.get_layer(producer_layer)
-    output_aux_node_name = PREFIX_AUXILIARY_OUTPUT_NODE + '_0'
-    node_attrs = {
-        NNCFGraph.NODE_TYPE_ATTR: NNCFGraphNodeType.OUTPUT_NODE,
-        NNCFGraph.METATYPE_ATTR: OutputNoopMetatype,
-        'original_name': output_aux_node_name
-    }
-    edge_attrs = {
-        NNCFGraph.ACTIVATION_SHAPE_EDGE_ATTR: _prepare_shape(output_model_layer.output_shape),
-        NNCFGraph.IN_PORT_NAME_EDGE_ATTR: 0,
-    }
-    nncf_graph.add_node(output_aux_node_name, **node_attrs)
-    nncf_graph.add_edge(producer_layer, output_aux_node_name, **edge_attrs)
+        nncf_node = nncf_graph.add_nncf_node(layer_name,
+                                             node_type=layer_type,
+                                             node_metatype=layer_metatype,
+                                             module_attributes=module_attributes,
+                                             layer_name=layer_name,
+                                             is_shared=False)
+
+        if producer_layer_id is not None:
+            input_shape = _prepare_shape(model.get_layer(layer_name).input_shape)
+            nncf_graph.add_edge_between_nncf_nodes(producer_layer_id,
+                                                   nncf_node.node_id,
+                                                   tensor_shape=input_shape[0],
+                                                   input_port_id=0,
+                                                   dtype=Dtype.FLOAT)  # TODO: determine from keras layers
```

Comment:
Please, add TODO according to the Style Guide.

Reply:
Done